lib/kits/core/request.js

```diff
@@ -1,129 +1,64 @@
 import fetch from 'cross-fetch';
 import SGN from '../../sgn-sdk';
 import {error, promiseCallbackInterop} from '../../util';
-import {ensure, saveToken, sign} from './session';
 
-function request(options = {}, callback, secondTime) {
-    ensure((err) => {
-        if (err != null) {
-            return callback(err);
-        }
-
-        let url = SGN.config.get('coreUrl') + (options.url ?? '');
-        const headers = options.headers ?? {};
-        const json = typeof options.json === 'boolean' ? options.json : true;
-        let token = SGN.config.get('coreSessionToken');
-        const appKey = SGN.config.get('appKey');
-        const appVersion = SGN.config.get('appVersion');
-        const appSecret = SGN.config.get('appSecret');
-        const locale = SGN.config.get('locale');
-        const qs = options.qs ?? {};
-        const geo = options.geolocation;
-        let {body} = options;
-
-        headers['X-Api-Key'] = appKey;
-        headers['X-Token'] = token;
-        if (appSecret != null) {
-            headers['X-Signature'] = sign(appSecret, token);
-        }
-
-        if (json) {
-            headers['Content-Type'] = 'application/json';
-            headers['Accept'] = 'application/json';
+function request(options = {}, callback) {
+    let url = SGN.config.get('coreUrl') + (options.url ?? '');
+    const method = options.method || 'get';
+    const headers = options.headers ?? {};
+    const appKey = SGN.config.get('appKey');
+    const clientVersion = SGN.config.get('clientVersion');
+    const body = options.body;
 
-            if (body) {
-                body = JSON.stringify(body);
-            }
-        }
-
-        if (locale != null) {
-            qs.r_locale = locale;
-        }
```

Comment:
Consider removing the accompanying `locale` stuff from the config code, this is the only place it has mattered.

Reply:
Good idea. Done + appSecret.